clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -20,9 +27,12 @@ import org.broadinstitute.clio.util.model.UpsertId
   */
 class ElasticsearchIndex[CI <: ClioIndex](
   val clioIndex: CI,
-  private[elasticsearch] val fieldMapper: ElasticsearchFieldMapper
+  private[elasticsearch] val defaultFields: Json = Json.obj(),
```

Comment:
Defaults usually go after non-defaults (otherwise you're forced to pass them to get to the trailing non-defaults, or use named arguments).